Formula/node.rb

```diff
@@ -19,7 +20,6 @@ class Node < Formula
 
   deprecated_option "enable-debug" => "with-debug"
 
-  depends_on "python@2" => :build if MacOS.version <= :snow_leopard
```

Comment:
We should leave these in the formula for now. I don't want start mass removing these wherever we think it may not actually work.

Reply:
see: https://github.com/Homebrew/homebrew-core/pull/24874#discussion_r172617118